packages/sdk/typescript/human-protocol-sdk/src/graphql/queries/operator.ts

```diff
@@ -26,22 +26,32 @@ const LEADER_FRAGMENT = gql`
 `;
 
 export const GET_LEADERS_QUERY = (filter: ILeadersFilter) => {
-  const { role } = filter;
+  const { role, roles, minAmountStaked } = filter;
 
   const WHERE_CLAUSE = `
     where: {
-      ${role ? `role: $role` : ''}
+      ${minAmountStaked ? `amountStaked_gt: $minAmountStaked` : ''}
+      ${roles ? `role_in: $roles` : ''}
```

Comment:
Should it be one or another for `role`/`roles`?

Ideally would be nice to keep just `roles`, so when you need one - you provide a single value. However it's a breaking change and we should not just remove `role` if we expect somebody else might be using it, but IMU:
- we are probably only consumers of this particular API
- it's used only on dashboard

So maybe we can safely remove `role` as for now, wdyt?

Reply:
Yes we can do it

Reply:
Yes, agree